package/network/services/openvpn/files/openvpn.init

```diff
@@ -69,6 +69,38 @@ section_enabled() {
 	[ $enable -gt 0 ] || [ $enabled -gt 0 ]
 }
 
+openvpn_get_dev() {
+	local dev
+	local dev_type
+	local name="$1"
+	local conf="$2"
+
+	# Do override only for configurations with config_file
+	config_get config_file "$name" config
+	[ -n "$config_file" ] || return
+
+	# Check there is someething to override
+	config_get dev "$name" dev
+	config_get dev_type "$name" dev_type
+	[ -n "$dev" ] || return
+
+	# If there is a no dev_type, try to guess it
+	if [ -z "$dev_type" ]; then
+		if grep -q '^[[:blank:]]*dev-type[[:blank:]]*tun' "$conf" || \
```

Comment:
Option values in native configuration may also be single or double quoted. Maybe we can reuse https://github.com/openwrt/openwrt/blob/master/package/network/services/openvpn/files/etc/hotplug.d/openvpn/01-user#L3 and use a logic similar to this one:

```
local dev dev_type
if get_option dev dev || get_option dev_type dev-type; then
    case "${dev_type:-$dev}" in
        tun*) echo "--dev-type tun --dev $dev" ;;
        ?*) echo "--dev-type tap --dev $tap" ;;
    esac
fi